css/css-ui/reference/transparent-accent-color-001-ref.html

```diff
@@ -4,38 +4,67 @@
 <link rel="help" href="https://drafts.csswg.org/css-ui-4/#widget-accent">
 <style>
 
-div {
-    border: solid orange;
-    padding: 1ch;
-    margin: 1ch;
-    float: left;
-}
-
-#t1 input { color-scheme: light;}
-#t1 input { accent-color: white; }
-
-#t2 input { color-scheme: light;}
-#t2 input { accent-color: #7f7fff; }
-
-#t3 { background: orange;}
-#t3 input { color-scheme: light;}
-#t3 input { accent-color: #7f7fff; }
-
+  div {
+      border: solid orange;
+      padding: 1ch;
+      margin: 1ch;
+      float: left;
+  }
+  
+  input { color-scheme: light;}
+  #extract-canvas{ accent-color: canvas; display:none;}
+  #t3 { background: orange;}
+  
 </style>
+  
+<p>Test passes if in each box below, you see a pair of identically colored check-boxes.
 
-<p>Test passes if in each box bellow, you see a pair of identically colored check-boxes.
-
-<div id=t1>
+<input id='extract-canvas' type=checkbox>
+<div id=t1 class="container">
   <input class=test type=checkbox checked>
   <input class=ref type=checkbox checked>
 </div>
 
-<div id=t2>
+<div id=t2 class="container">
   <input class=test type=checkbox checked>
   <input class=ref type=checkbox checked>
 </div>
 
-<div id=t3>
+<div id=t3 class="container">
   <input class=test type=checkbox checked>
   <input class=ref type=checkbox checked>
 </div>
+
+<script>
+
+  var testColors = [[255,0,0,0], [0,0,255,0.5], [0,0,255,0.5]]
+
+  function getColorComponents(color){
+    var components = []
+    color.substring(color.indexOf("(") + 1).replace(/\s/g,"").split(",").forEach((s) => {
+      components.push(parseFloat(s));
+    });
+    return components;
```

Comment:
```suggestion
    return color.substring(color.indexOf("(") + 1).replace(/\s/g,"").split(",").map(parseFloat);
```

Array.map makes this shorter